.travis.yml

```diff
@@ -6,11 +6,7 @@ php:
 
 # list of paths/bundle to execute
 env:
-    # run setup wizard to install Demo+Content and then test demo with content
   - INSTALL="demoContentNonUniqueDB" PROFILE="demo" TEST="content"
```

Comment:
Well, I kind of want to be sure that the DemoBundle works as expected. Tests are one way to get started, but I'm happily deleting those that don't make sense.

I'll take the simplest path, but I'm afraid that I'm slowed down by the remaining legacy requirements in the demobundle... (unless i'm missing something).